index.js

```diff
@@ -47,25 +54,30 @@ module.exports = (ctx, req, res) => {
         return end(500, { message: 'Error authenticating to the Auth0 API.', details: err.message });
       }
 
-      var connectionUrl = `https://${ctx.data.AUTH0_DOMAIN}/api/connections/${ctx.data.connection}`;
-      var monitorUrl = `https://${ctx.data.AUTH0_DOMAIN}/api/connections/${ctx.data.connection}/socket`;
-      console.log('Monitoring:', monitorUrl);
-
+      const connectionsUrl = `https://${domain}/api/v2/connections`;
       request.get({
-        url: connectionUrl,
+        url: connectionsUrl,
         headers: {
           'Authorization': `Bearer ${token}`
-        }
+        },
+        qs: {
```

Comment:
Add `fields: "id,name,strategy"` so we dont bring the whole object and the expensive `enabled_clients`

Reply:
Good catch :) Fixed, did not asked for name though.